build_tools/packaging/python/templates/rocm/src/rocm_sdk/_dist_info.py

```diff
@@ -158,19 +159,20 @@ def determine_target_family() -> str:
 )
 
 # Public libraries.
-LibraryEntry("amdhip64", "core", "libamdhip64.so.6")
-LibraryEntry("hiprtc", "core", "libhiprtc.so.6")
-LibraryEntry("roctx64", "core", "libroctx64.so.4")
-LibraryEntry("rocprofiler-sdk-roctx", "core", "librocprofiler-sdk-roctx.so.1")
-
-LibraryEntry("hipblas", "libraries", "libhipblas.so.3")
-LibraryEntry("hipfft", "libraries", "libhipfft.so.0")
-LibraryEntry("hiprand", "libraries", "libhiprand.so.1")
-LibraryEntry("hipsparse", "libraries", "libhipsparse.so.4")
-LibraryEntry("hipsolver", "libraries", "libhipsolver.so.1")
-LibraryEntry("rccl", "libraries", "librccl.so.1")
-LibraryEntry("hipblaslt", "libraries", "libhipblaslt.so.1")
-LibraryEntry("miopen", "libraries", "libMIOpen.so.1")
+LibraryEntry("amdhip64", "core", "libamdhip64.so.6", "amdhip64_6.dll")
+LibraryEntry("hiprtc", "core", "libhiprtc.so.6", "hiprtc0605.dll")
+LibraryEntry("roctx64", "core", "libroctx64.so.4", "")
+LibraryEntry("rocprofiler-sdk-roctx", "core", "librocprofiler-sdk-roctx.so.1", "")
+
+LibraryEntry("amd_comgr", "core", "libamd_comgr.so.3", "amd_comgr0605.dll")
+LibraryEntry("hipblas", "libraries", "libhipblas.so.3", "")
```

Comment:
Tested with PyTorch, need `libhipblas.dll` here. Missed that in the alphabetical list because of course one of the subprojects uses a different library name.